### ISSUE TITLE
Telegraf inputs.snmp plugin - add support for converting decimal index to ASCII text

### ISSUE THREAD

**Issue by svestenik** [influxdata/telegraf#11733]
Hi team,
I am looking for a way for Telegraf SNMP Input plugin to convert decimal index into ASCII text. Searching the channel on this topic is not producing anything useful.
Example:
When I poll the device in question (Cisco router) via NetSNMP from CLI I get the following result:
`snmpget -v2c -c somecommunity 123.123.123.123 MPLS-VPN-MIB::mplsVpnVrfRouteDistinguisher`

output:
`MPLS-VPN-MIB::mplsVpnVrfRouteDistinguisher."internet" = STRING: "65500:40104"`
Where "internet" is the name of Vrf and string result is route descriptor.

When I use Telegraf for the same query:

```
[[inputs.snmp.table]]
   name = "VrfRDTable"
#   oid = "MPLS-VPN-MIB::mplsVpnVrfTable"
   inherit_tags = ["device"]
   index_as_tag = true

[[inputs.snmp.table.field]]
   name = "VrfRouteDistinguisher"
   oid = "MPLS-VPN-MIB::mplsVpnVrfRouteDistinguisher"
   is_tag = true

[[inputs.snmp.table.field]]
   name = "VrfDescription"
   oid = "MPLS-VPN-MIB::mplsVpnVrfDescription"
```

I get the following:

VrfRDTable,index=8.105.110.116.101.114.110.101.116.1.3,VpnVrfRouteTarget="65500:40104"

Here, we see a decimal representation of VRF name as index. 

Cisco documentation on this states as follows:

: The mplsVpnVrfName is actually an octet string that represents the string length (4) and the ASCII codes for each character.
For example, "VPN1" is represented as "4.86.80.78.49".

This practice is widely used for SNMP OID's throught networking industry, so I think it would be beneficial to add some way of translating decimal to ASCII string.

**Comment by Hipska** [comment]
Hi, we discussed this on Slack where I gave you a solution by using the Starlark processor. So I added labels for this being a feature request, as indeed it's like this a lot in the snmp world.

**Comment by svestenik** [comment]
> Hi, we discussed this on Slack where I gave you a solution by using the Starlark processor. So I added labels for this being a feature request, as indeed it's like this a lot in the snmp world.

Exactly, and I can confirm that Starlark processor script partly working. There is still a bit of coding on the Starlark processor required for full support. But, I think that this needs to be handled by SNMP plugin in full, since coding in Starlark is not everybody's strength :).

**Comment by Hipska** [comment]
We could make a generic starlark script that works with variables to specify what you want to convert to where..

**Comment by svestenik** [comment]
> We could make a generic starlark script that works with variables to specify what you want to convert to where..

That could work.
